src/osdc/Objecter.cc

```diff
@@ -3569,6 +3573,8 @@ void Objecter::handle_osd_op_reply(MOSDOpReply *m)
   ldout(cct, 15) << "handle_osd_op_reply completed tid " << tid << dendl;
   _finish_op(op, 0);
 
+  logger->tinc(l_osdc_op_latency, ceph::coarse_mono_time::clock::now() - op->stamp);
```

Comment:
regarding the valgrind issues, it may be that `_finish_op(op, 0);` is freeing this `op`?

Reply:
you are right. see line 3185. the memory is released. thanks!